.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: c
+dist: bionic
+compiler: gcc
+before_install:
+  - src=$(pwd)
+  - mkdir /tmp/linux
+  - cd /tmp/linux
+  - git init
+  - git remote add origin git://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git
+  - git fetch --depth 1 origin e91ef5bcdeda8956eb9f1972ed90198b698dca0f # 5.2.21
```

Comment:
It's cleaner to use the tag `v5.2.21` than the SHA.

Reply:
I thought only SHA1 worked (from reading some online doc) and never tried the tag. Tried it now separately and it works, so I'll do that.